common/common.go

```diff
@@ -354,3 +355,24 @@ func SplitStringSliceByLimit(in []string, limit uint) [][]string {
 	}
 	return sliceSlice
 }
+
+// In_array checks if _val_ belongs to _array_
+func In_array(val interface{}, array interface{}) (exists bool, index int) {
```

Comment:
This needs a test function also as per our coding style, function names use pascal case; so could you please change name to InArray.

Reply:
agreed and done. I am not sure for InArray to return a position of the value in case of a Slice, as I noted the position might change depending on whether an initial size length is set. Do you think it would be best allowing slices as inputs to InArray?